diff_drive_control_node:
  ros__parameters:
    wheel_radius: 0.1
    robot_width: 0.5
    encoder_ticks_per_rev: 1000
    max_motor_speed: 200.0
    control_motor_frequency: 10.0
    control_odometry_frequency: 10.0
    odom_frame_id: "odom"
    base_frame_id: "base_link"
    odom_topic: "odom"
    publish_tf: true

cmd_vel_filter_node:
  ros__parameters:
    acceleration_limit: 0.7
    deceleration_limit: 2.5
    publish_rate: 20.0
    cmd_vel_timeout: 0.1
